examples/failing-examples/bound-vs-free.golden

```diff
@@ -0,0 +1,4 @@
+Module not found: do.kl
```

Comment:
This test is failing for the wrong reason! This example should fail with the user-defined error message "variables must be distinct".

Reply:
Yea I saw that too. I wonder if the testrunner does having the right KLISTERPATH.